dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/categories/general/subcategories/LksgProductionSpecific.kt

```diff
@@ -4,7 +4,6 @@ import com.fasterxml.jackson.annotation.JsonProperty
 import org.dataland.datalandbackend.model.enums.commons.YesNo
 import org.dataland.datalandbackend.model.enums.lksg.NationalOrInternationalMarket
 import org.dataland.datalandbackend.model.lksg.LksgProductionSite
-import java.math.BigDecimal
 
```

Comment:
(Together with `LksgProductionSpecific`)

Reply:
Done

Reply:
I just noticed that we could do the same kind of structuring also in the `sfdr` folder.
It also has the categories `environmental` and `social`, and also some `subcategories` to each one of them.
Would be nice in my opinion.

Reply:
But it is kind of out of scope for this story. Maybe make a note to the next SFDR story in the backlog?

Reply:
Agree in general, but in this case with IntelliJ it should take only a minute to refactor that. The IDE takes care of the rest.

Of course, if there is more trouble to it, let's postpone it.

Reply:
done

Reply:
Nice, thanks.
Could you create a note (in the ticket for the sfdr form creation maybe?) that we also put the `SfdrGeneral` sub-model directly into the `SfdrData` main model, instead of nesting it under `SfdrSocial` ?
This would then be consistent with Lksg, and I also think it makes sense.
What do you think?

Reply:
Agreed, I put a comment into the SFDR refactoring story

Reply:
Perfect. Thanks. Closing this.